controllers/articles.controllers.js

```diff
@@ -1,4 +1,11 @@
-const {selectArticleById} = require('../models/articles.models')
+const {selectArticles, selectArticleById} = require('../models/articles.models')
+
+exports.sendArticles = (req, res) => {
+    console.log('in the controller')
```

Comment:
We should remove console.logs 🙂

Reply:
oops forgot this one, feeling a bit better thanks! still improving